lib/Migration/Version00004Date20200428102743.php

```diff
@@ -37,11 +37,11 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		]);
 		$table->addColumn('client_id', 'string', [
 			'notnull' => true,
-			'length' => 64,
+			'length' => 2048,
```

Comment:
For fresh installations of the app, all the migration steps will run so it's not necessary to change the old one.